sygnal/helper/proxy/proxy_asyncio.py

```diff
@@ -0,0 +1,241 @@
+import asyncio
```

Comment:
This could do with a copyright notice. It's hazy as to whether it should be matrix foundation or NVL, but I think you're doing this with a matrix.org hat on so matrix foundation.

Reply:
(likewise other new files that don't have a notice)